test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -103,6 +134,42 @@ class AssetListPage {
     return assets.length;
   }
 
+  async getTokenListNames(): Promise<string[]> {
+    console.log(`Retrieving the list of token names`);
+    const tokenElements = await this.driver.findElements(this.tokenListItem);
+    const tokenNames = await Promise.all(
+      tokenElements.map(async (element) => {
+        return await element.getText();
+      }),
+    );
+    return tokenNames;
+  }
+
+  async getAssetPercentageIncreaseDecrease(
```

Comment:
It seems we should remove this function?  as you have made another implementation to address the comment. Since this function is no longer used, it's better to remove it to prevent people from using it, as it follows an anti-pattern.

Reply:
good spot